modules/cli/pages/cli-intro.adoc

```diff
@@ -47,6 +47,8 @@ Download the command line tools package for your platform through the following
 
 * https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-windows_amd64.zip[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-windows_amd64.zip]
 
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_arm64.zip[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_arm64.zip]
+
```

Comment:
I realize that this is incredibly picky, but could the two macos packages be listed one after the other?  Mainly because I don't want someone just seeing the first one and picking up the wrong one.  Thanks.   I'll go ahead and approve this review.

Reply:
No problem. I've modified. Thanks.